src/app.py

```diff
@@ -78,7 +84,29 @@ def bootstrap(_return=0):
 
                 res = requests.get(url=master_url)
                 if res.ok:
-                    logger.info('Start load test automatically for {duration} seconds.'.format(duration=duration))
+                    timeout = time.time() + SLAVES_WAITING_TIME
+                    connected_slave = 0
+                    while time.time() < timeout:
+                        try:
+                            logger.info('Checking if all slave(s) are connected.')
+                            stats_url = '/'.join([master_url, 'stats/requests'])
+                            res = requests.get(url=stats_url)
+                            connected_slave = res.json().get('slave_count')
+
+                            if connected_slave == total_slaves:
+                                break
+                            else:
+                                logger.info('Current connected slave: {con}'.format(con=connected_slave))
+                                time.sleep(SLAVES_SLEEP_TIME)
+                        except ValueError as v_err:
+                            logger.error(v_err.message)
+                    else:
```

Comment:
Actually I'm not sure if we should abort the test in this case. IMO it would be better to print a warning and let it run.